synapse/tests/test_lib_storm.py

```diff
@@ -862,15 +862,25 @@ async def test_lib_storm_basics(self):
             pkg1 = {'name': 'haha', 'version': '1.2.3'}
             await core.addStormPkg(pkg1)
             msgs = await core.stormlist('pkg.list')
-            self.isin('haha', msgs[2][1]['mesg'])
-            self.isin('hehe', msgs[3][1]['mesg'])
+            self.isin('haha', msgs[3][1]['mesg'])
```

Comment:
We should be able to replace the individual item checks with `self.stormIsInPrint('some message', msgs)` and the test should be less brittle in the future :+1: